mod/cp_notifications/start.php

```diff
@@ -925,7 +998,9 @@ function cp_create_notification($event, $type, $object) {
 			// the user creating the content is automatically subscribed to it
 			add_entity_relationship(elgg_get_logged_in_user_guid(), 'cp_subscribed_to_email', $object->getGUID());
 			add_entity_relationship(elgg_get_logged_in_user_guid(), 'cp_subscribed_to_site_mail', $object->getGUID());
-			break;
+			break; 
+
+		case 'single_file_upload':
```

Comment:
Empty case? If it's supposed to fall through, then maybe a comment `// falls through` or whatever the PHP version is